usd_convert: 0.6
eur_convert: 3.1
gbr_convert: 0.1
ugx_convert: 5.0
